src/main/resources/templates/languages/en/non-strategic/interimApplicationsChdDailyCauseList.json

```diff
@@ -0,0 +1,19 @@
+{
+  "listName": "Interim Applications List (Chancery Division) Daily Cause List",
+  "venueLine1": "Business and Property Courts Rolls Building",
```

Comment:
Similar with the style guide, the venue name should be like the style guide, and the first line of the venue in bold